src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/VtableIndexStubGenerator.cs

```diff
@@ -188,11 +220,31 @@ private static SyntaxTokenList StripTriviaFromModifiers(SyntaxTokenList tokenLis
                 }
                 implicitThis = (bool)implicitThisValue.Value!;
             }
+            if (namedArguments.TryGetValue(nameof(VirtualMethodIndexData.ExceptionMarshalling), out TypedConstant exceptionMarshallingValue))
+            {
+                // TypedConstant's Value property only contains primitive values.
+                if (exceptionMarshallingValue.Value is not int)
```

Comment:
Comment says "only contains primitive values", but we only check for `int`. Is this accurate?

Reply:
Yes, this is accurate. Basically `typedConstant.Value is EnumType` will always be `false`, even though you could cast `(EnumType)typedConstant.Value` if `typedConstant.Value` is an `int`.